pages/linux/lrzuntar.md

```diff
@@ -0,0 +1,24 @@
+# lrzuntar
+
+> A wrapper for `lrunzip` to simplify decompression of directories.
+> See also: `lrztar`, `lrzip`.
+
+- Decompress from file to a directory:
+
+`lrzuntar {{path/to/directory.tar.lrz}}`
+
+- Decompress from file to a directory using a specific number of processor threads:
+
+`lrzuntar -p {{8}} {{path/to/directory.tar.lrz}}`
+
+- Decompress from file to a directory and silently overwrite items that already exist:
+
+`lrzuntar -f {{directory.tar.lrz}}`
+
+- Specify output path:
```

Comment:
```suggestion
- Specify the output path:
```

Also, `directory.tar.lrz` -> `path/to/output_archive.tar.lrz`?

Reply:
The output path in this instance is the directory that the decompressed directory will appear within.

Reply:
Ah, ok.